bitrise.yml

```diff
@@ -0,0 +1,532 @@
+            - team_id: "$DEVELOPER_TEAM"
+            - git_url: git@github.com:nodes-projects/internal-certificates-ios.git
+      - flutter-installer@0: {}
+      - script@1:
+          title: Flutter Pub Get
+          inputs:
+            - content: flutter pub get
+      - flutter-build@0:
+          inputs:
+            - ios_additional_params:
+                " --release --no-codesign -t lib/main_production.dart
+                --flavor production --build-number=$BITRISE_BUILD_NUMBER"
+            - cache_level: none
+            - android_output_type: appbundle
+            - android_additional_params:
+                "--release -t lib/main_production.dart --flavor
+                production --build-number=$BITRISE_BUILD_NUMBER"
+      - set-env-var@0:
+          inputs:
+            - value: "$BUNDLE_ID_PRODUCTION"
+            - destination_keys: APP_BUNDLE_ID
+      - xcode-archive@3:
+          inputs:
+            - configuration: Release-production
+            - export_method: ad-hoc
+            - team_id: "$DEVELOPER_TEAM"
+            - scheme: production
+      - deploy-to-bitrise-io@1: {}
+      - script@1:
+          inputs:
+            - content: |-
+                #!/bin/bash
+
+                input=${BITRISE_PUBLIC_INSTALL_PAGE_URL_MAP}
+                inputs=($(echo "$input" | tr '|' '\n'))
+
+                for i in "${inputs[@]}"
+                do
+                  if [[ "$i" == *".apk"* ]]; then
+                    androidLink=${i##*=>}
+                  elif [[ "$i" == *".ipa"* ]]; then
+                    iosLink=${i##*=>}
+                  else
+                    echo "No install page link found."
+                  fi
+                done
+
+                echo "android: $androidLink"
+                echo "ios: $iosLink"
+
+                envman add --key ANDROID_INSTALL_PAGE_LINK --value "${androidLink}"
+                envman add --key IOS_INSTALL_PAGE_LINK --value "${iosLink}"
+          title: Set Install Page Links
+      - script@1:
+          title: Set Build Version
+          inputs:
+            - content: |-
+                #!/usr/bin/env bash
+
+                input=`grep -w version: pubspec.yaml`
+                inputs=($(echo "$input" | tr 'version:' '\n'))
+
+                version="${inputs[@]}"
+
+                echo "BUILD_VERSION: ${version}"
+
+                envman add --key BUILD_VERSION --value "${version}"
+      - script@1:
+          title: Set Build Flavour
+          inputs:
+            - content: envman add --key BUILD_FLAVOUR --value "PRODUCTION"
+      - google-play-deploy@3:
+          inputs:
+            - package_name: "$BUNDLE_ID_PRODUCTION"
+            - release_name: "${BUILD_VERSION} (${BITRISE_BUILD_NUMBER})"
+            - service_account_json_key_path: "$BITRISEIO_GOOGLE_PLAY_SERVICE_ACCOUNT_KEY_URL"
+    after_run:
+      - deployTestflight
+      - notifySlack
+      - tag
+  deployTestflight:
+    steps:
+      - fastlane-match@0.2:
```

Comment:
@johsoe i would suggest going with something that we already tested and know that it works flawlessly rather than implementing the cloud signing thing right now on the template, we can re-add it later once we test it in development/production